pdfminer/converter.py

```diff
@@ -87,9 +95,10 @@ def paint_path(self, gstate, stroke, fill, evenodd, path):
                 line = LTLine(gstate.linewidth, (x0, y0), (x1, y1), stroke,
                               fill, evenodd, gstate.scolor, gstate.ncolor)
                 self.cur_item.add(line)
+                return
```

Comment:
Use if-elif-else statements instead of early returns

Reply:
Sure, done! Required some reordering of the logic, but in a good way (I think?).

Reply:
I reorderdered the if-selse staements once more. Let me know what you think.